components/dashboard/src/components/Menu.tsx

```diff
@@ -4,83 +4,191 @@
+        if (!showTeamsUI || !teams) {
+            return;
+        }
+        (async () => {
+            const members: Record<string, TeamMemberInfo[]> = {};
+            await Promise.all(teams.map(async (team) => {
+                const infos = await getGitpodService().server.getTeamMembers(team.id);
+                members[team.id] = infos;
+            }));
+            setTeamMembers(members);
+        })();
+    }, [ teams ]);
+
+    const leftMenu = (!!team
+        ? [
+            {
+                title: 'Projects',
+                link: `/${team.slug}/projects`,
+                alternatives: [`/${team.slug}`]
+            },
+            {
+                title: 'Members',
+                link: `/${team.slug}/members`
+            }
+        ]
+        : [
+            {
+                title: 'Workspaces',
+                link: '/workspaces',
+                alternatives: ['/']
+            },
+            {
+                title: 'Settings',
+                link: '/settings',
+                alternatives: settingsMenu.flatMap(e => e.link)
+            }
+        ]
+    );
+    const rightMenu = [
+        ...(user?.rolesOrPermissions?.includes('admin') ? [{
+            title: 'Admin',
+            link: '/admin',
+            alternatives: adminMenu.flatMap(e => e.link)
+        }] : []),
+        {
+            title: 'Docs',
+            link: 'https://www.gitpod.io/docs/',
+        },
+        {
+            title: 'Community',
+            link: 'https://community.gitpod.io/',
+        }
+    ];
+
+    return <>
+        <header className="lg:px-28 px-10 flex flex-col pt-4 space-y-4">
+            <div className="flex">
+                <div className="flex justify-between items-center pr-3">
+                    <Link to="/">
+                        <img src={gitpodIcon} className="h-6" />
+                    </Link>
+                    <div className="ml-2 text-base">
+                        {showTeamsUI
+                            ? <ContextMenu classes="w-64 left-0" menuEntries={[
+                                {
+                                    title: userFullName,
+                                    customContent: <div className="w-full text-gray-400 flex flex-col">
+                                        <span className="text-gray-800 text-base font-semibold">{userFullName}</span>
+                                        <span className="">Personal Account</span>
+                                    </div>,
+                                    separator: true,
+                                    onClick: () => history.push("/"),
+                                },
+                                ...(teams || []).map(t => ({
+                                    title: t.name,
+                                    customContent: <div className="w-full text-gray-400 flex flex-col">
+                                        <span className="text-gray-800 text-base font-semibold">{t.name}</span>
```

Comment:
**issue**: Reminder to add colors for dark theme and use `text-sm` font size!

```suggestion
                                        <span className="text-gray-800 text-sm dark:text-gray-300 font-semibold">{t.name}</span>
```

Reply:
@gtsiolis: The [spec](https://www.figma.com/file/24YZdNs7MmS5drlZ5ZlM7r/Allow-adding-and-switching-teams-%234355?node-id=617%3A179) currently has `text-base` a.k.a. `16px` as font size for team names in the team switcher-menu.

Reply:
Now tracked in https://github.com/gitpod-io/gitpod/issues/4471